### ISSUE TITLE
Error in the import and synchronization module

### ISSUE THREAD

**Issue by maxlinux2000** [Chocobozzz/PeerTube#5425]
### Describe the current behavior

I have 2 youtube playlist. One can be imported and the other cannot.
This occurs using the import on a Peertube installed today (v. 4.3.1 on debian 11 on amd64)

I have created a test channel and tried to import a small playlist from this url
https://youtu.be/7ppQVOKuKzA
and this works perfectly

Then I have created another test channel and trying to import another play list from this url:
https://youtu.be/5jcS_DlhbiM
And this gives an error that can be seen in the log below.

I have repeated the experiment with another instance of peertube on yunohost on raspberryPi4, and it gives exactly the same error.

### Steps to reproduce

1. create a channel 
2. add a synchronization using this url  https://youtu.be/5jcS_DlhbiM 

Result: Fail

![Captura de pantalla de 2022-11-22 20-52-35](https://user-images.githubusercontent.com/5683921/203413697-acd95946-dbed-4ec2-83ec-dc4f4efa7d8e.png)
![Captura de pantalla de 2022-11-22 20-52-50](https://user-images.githubusercontent.com/5683921/203413703-90e3211f-ece4-42dc-91d8-ee6c5f487821.png)
![Captura de pantalla de 2022-11-22 20-53-00](https://user-images.githubusercontent.com/5683921/203413706-3dc96357-20b2-41bd-98a5-af5bc16c3797.png)

Here the log in /var/www/peertube/storage/logs/peertube.log

https://paste.yunohost.org/osadopikow.pl

### Describe the expected behavior

what I hope is that you can import any channel or playlist to synchronize channels

### Additional information

* PeerTube instance: local
  * URL: intranet, not public
  * Version: 4.3.1
  * NodeJS version: v12.22.5
  * Ffmpeg version: 4.3.4-0+deb11u1

* Browser name, version and platforms on which you could reproduce the bug: Firefox ESR 91
* Link to browser console log if relevant: none
* Link to server log if relevant (`journalctl` or `/var/www/peertube/storage/logs/`):  https://paste.yunohost.org/osadopikow.pl

**Comment by maxlinux2000** [comment]
Please search in the log this

![Captura de pantalla de 2022-11-22 21-43-21](https://user-images.githubusercontent.com/5683921/203417572-350e882a-6019-4154-b652-87ece106afdc.png)

**Comment by Chocobozzz** [comment]
Hello,

Please don't paste a screen of logs but copy/paste the characters instead.
I'm also not sure to understand the issue because you're talking about playlist but the youtube URLs are videos.

**Comment by maxlinux2000** [comment]
Hi the log is here

https://paste.yunohost.org/osadopikow.pl

I'm trying with another fresh installation downloading two channels.... for the moment no problem.

I've tried before with playlist because youtube-dl can download them and are a few videos.

I will try tomorrow with a chemical channel. I think that we will have errors because peertube can't do the login an a lot of videos are obscured if you are not logged.
I have solved this problem in my downloader script using the file cookies.txt obtained from a firefox extension.
Using the cookies of firefox,  youtube-dl can simulate the login and download all.
But the peertube downloader is very confortable and easy to use
